packages/ui-library/src/components/checkbox/index.test.ts

```diff
@@ -34,6 +34,23 @@ describe('blr-checkbox', () => {
     expect(className).to.contain('input-control');
   });
 
+  it('has error Icon is set to undefined', async () => {
```

Comment:
Please adjust it to it('has error Icon set to undefined'...) [remove 'is' as we discussed yesterday]